### ISSUE TITLE
Search results page -- make facets' disclosure arrow keyboard accessible

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#149]
From NYU A11y Review:  (issue #4 part 2) 

The element that signifies that the Publications menu has more options:

<i class="angle right icon"></i>

is not actionable and not associated with the checkbox. Also, the sub-filter elements cannot be selected unless the Publications element is selected. Consider removing the "show/hide" design of this element so that all options are exposed.
